.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    paths:
+      - "projects/**"
+      - "tests/**"
```

Comment:
Here, we might want to consider other paths to include for rebuilding the image; e.g. `**/requirements.txt` and `scripts/**`.
Another option would be to omit this part to rebuild on every push.
tag: @glatard